ctapipe/tools/stage1.py

```diff
@@ -64,10 +70,15 @@ class Stage1Tool(Tool):
             {"Stage1Tool": {"progress_bar": True}},
             "show a progress bar during event processing",
         ),
+        "analyse-muons": (
```

Comment:
I suggest using the `write_muons` parameter of DL1DataWriter here,  to be symmetric with how the other options are specified.  We don't have options like "analyze_parameters". I.e. don't add a parameter here, but use the one from the writer (see how it is done in the loop for image parameters).

We can rethink the options later, but I would like to keep them consistent for now.

Reply:
I like `analyze-muons` better, since it is not only writing that is enabled. More calculations are performed.

The writer only writes, it doesn't decided whether the parameters are calculated or not.

Reply:
I agree, however right now I would like to be consistent with the other options, at least for this minor relase.  For the next major release (containing DL2 info) I am already in the process of re-designing all of the options to be consistent, and there there will be a separation between computing and writing that is clear, but that is really a separate issue that should not be solved here.